extensions/cache/runtime/src/main/java/io/quarkus/cache/runtime/caffeine/CaffeineCacheImpl.java

```diff
@@ -122,6 +124,26 @@ public CompletableFuture<Object> apply(Object key) {
                 }).map(fromCacheValue());
     }
 
+    @Override
+    public <K, V> V compute(K key, BiFunction<K, V, V> command) {
```

Comment:
@ben-manes this is the compute method I am trying to implement on top of Caffeine. it is similar to the compute method in the caffeine api without the retry